test/components/pagination/pagination.test.js

```diff
@@ -0,0 +1,78 @@
+import '../../../src/components/pagination/pager-numeric.js';
+import { expect, fixture, html, oneEvent } from '@brightspace-ui/testing';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { runConstructor } from '@brightspace-ui/core/tools/constructor-test-helper.js';
+
+const custompageSizes = [2, 5, 37, 159];
+
+async function createComponent({ pageNumber, maxPageNumber, showPageSizeSelector = false, pageSizes, pageSize } = {}) {
+	return await fixture(html`<d2l-labs-pager-numeric
+		page-number="${ifDefined(pageNumber)}"
+		max-page-number="${ifDefined(maxPageNumber)}"
+		?show-page-size-selector="${showPageSizeSelector}"
+		.pageSizes="${ifDefined(pageSizes)}"
```

Comment:
Nit: I think `ifDefined` isn't really valid for properties, since it's an attribute directive. I think Lit might tolerate it though. I suspect we can just remove the `ifDefined(...)` here if we're using the property. Or, if we prefer to use the attribute, then we could use `ifDefined(pageSizes ? JSON.stringify(pageSizes) : undefined)`.